Elpis/MainWindow.xaml.cs

```diff
@@ -71,7 +60,7 @@ public partial class MainWindow : Window
 
         private string _configLocation;
 
-        private Config _config;
+        private readonly Config _config;
```

Comment:
Sure it will - it is not possible to assign _config field itself outside of constructor.